src/UXClient/Components/Legend/Legend.ts

```diff
@@ -29,9 +29,9 @@ class Legend extends Component {
                                 .style("width", (this.legendWidth) + "px"); // - 16 for the width of the padding
     }
 
-    private labelMouseoutWrapper (labelMouseout, svgSelection) {
+    private labelMouseoutWrapper (labelMouseout, svgSelection, event) {
         return (svgSelection, aggKey) => {
-            d3.event.stopPropagation();
+            event?.stopPropagation();
```

Comment:
nit: i think we can assume that the `event` has this method

Reply:
I added the check just in case undefined is passed, because the parent method `draw` has the event as optional.